sql/src/main/java/org/opensearch/sql/sql/domain/SQLQueryRequest.java

```diff
@@ -116,10 +138,6 @@ private boolean isOnlySupportedFieldInPayload() {
     return SUPPORTED_FIELDS.containsAll(jsonContent.keySet());
   }
 
-  private boolean isFetchSizeZeroIfPresent() {
```

Comment:
fetch_size must be 0?

Reply:
No. This was for the new SQL engine. It used to can't handle scroll request and should redirect it to the legacy engine.
Now I remove this limitation so that the new engine handles scroll requests.